tests/FVTTests/homeView/testResizingOfDatasetTree.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
```

Comment:
Only IBM Employees are required to add the IBM copyright statement to new files they create, You should consult with your own company if they require a specific header, the default Zowe one however is: `Copyright Contributors to the Zowe Project.`

Reply:
Yes, sorry, I copied another file as a template and missed that part. Changed to `Copyright Contributors to the Zowe Project.`. However it is a bit odd that part of code edited by the community is copyrighted by IBM.